src/test/java/uk/gov/companieshouse/presenter/account/kafka/KafkaMessageHelperTest.java

```diff
@@ -0,0 +1,133 @@
+import uk.gov.companieshouse.presenter.account.exceptionhandler.KafkaMessageFailException;
+import uk.gov.companieshouse.presenter.account.exceptionhandler.KafkaMessageInterruptedException;
+import uk.gov.companieshouse.presenter.account.serialiser.PresenterCreatedSerialiser;
+import uk.gov.companieshouse.presenter.account.utils.DateTimeGenerator;
+import uk.gov.companieshouse.presenter_account.PresenterCreated;
+
+@ExtendWith(MockitoExtension.class)
+class KafkaMessageHelperTest {
+
+    @Mock
+    private CHKafkaProducer kafkaProducer;
+
+    @Mock
+    DateTimeGenerator dateTime;
+
+    @Mock
+    private PresenterCreatedSerialiser serialiser;
+
+    @Mock
+    private Logger logger;
+
+    private KafkaMessageHelper messageHelper;
+
+    private static final String ID = "id";
+
+    private static final String TOPIC = "topic";
+
+    private static final byte[] VALUE = new byte[] { 0 };
+
+    private static final LocalDateTime TIME = LocalDateTime.of(2100, 1, 1, 1, 1);
+
+    @BeforeEach
+    void BeforeEach() {
+        messageHelper = new KafkaMessageHelper(kafkaProducer,
+                dateTime, serialiser, logger);
+    }
+
+    @Test
+    @DisplayName("Create presenter created with id")
+    void testCreatePresenterCreated() {
+        var presenterCreated = messageHelper.createPresenterCreated(ID);
+        assertEquals(ID, presenterCreated.getId());
+    }
+
+    @Test
+    @DisplayName("Create presenter created with null id")
+    void testCreatePresenterCreatedWithNullId() {
+        assertThrows(NullPointerException.class, () -> messageHelper.createPresenterCreated(null));
+    }
+
+    @Test
+    @DisplayName("Serialize presenter-created")
+    void testSerializePresenterCreated() {
+        PresenterCreated presenterCreated = mock(PresenterCreated.class);
+        when(serialiser.serialize(presenterCreated)).thenReturn(VALUE);
+        assertEquals(VALUE, messageHelper.serializePresenterCreated(presenterCreated));
+        verify(serialiser, times(1)).serialize(presenterCreated);
+    }
+
+    @Test
+    @DisplayName("Create message")
+    void testCreateMessage() {
+        when(dateTime.currentDateTime()).thenReturn(TIME);
+        Message message = messageHelper.createMessage(TOPIC, VALUE);
+        assertEquals(TOPIC, message.getTopic());
+        assertEquals(VALUE, message.getValue());
+        assertEquals(TIME.toEpochSecond(ZoneOffset.UTC), message.getTimestamp().longValue());
+    }
+
+    @Test
+    @DisplayName("Create message with blank topic")
+    void testCreateMessageWithBlankTopic() {
+        String blankTopic = "  ";
+        assertThrows(IllegalArgumentException.class, () -> {
+            messageHelper.createMessage(blankTopic, VALUE);
+        });
+    }
+
+    @Test
+    @DisplayName("Sent presenter create message")
+    void testSendPresenterCreateMessage() {
+        Message message = mock(Message.class);
+        Future<RecordMetadata> messageFuture = mock(Future.class);
```

Comment:
Again these don't need to be mocks, as we aren't mocking any of there functionality